src/lib/serviceRoutines/postSubscribeContext.cpp

```diff
@@ -72,7 +72,7 @@ std::string postSubscribeContext
   if (ciP->servicePathV.size() > 1)
   {
     char  noOfV[STRING_SIZE_FOR_LONG + 3];
-    snprintf(noOfV, sizeof(noOfV) - 1, "%lu", ciP->servicePathV.size());
+    snprintf(noOfV, sizeof(noOfV) - 1, "%lu", static_cast<unsigned long>(ciP->servicePathV.size()));
```

Comment:
Same same, normal typecast if possible.
At least, we need to find a way to do this that doesn't require C++ constructs